game/templates/game/register.html

```
@@ -11,12 +11,27 @@
+            try { savedPageTheme = localStorage.getItem('pageTheme') || 'dark'; } catch (e) { }
+            if (!VALID_PAGE_THEMES.includes(savedPageTheme)) savedPageTheme = 'dark';
+            document.documentElement.setAttribute('data-page-theme', savedPageTheme);
+        })();
+    </script>
 </head>
   <body>
        <!-- Back-to-home arrow -->
        <a href="{% url 'landing' %}" class="back-home" aria-label="Back to home">&larr; Home</a>
 
+       <!-- Theme Toggle Button -->
+       <button id="themeToggleBtn" class="theme-toggle-btn" title="Toggle Day/Night" aria-label="Toggle Day/Night mode">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)